example/.gitignore

```diff
@@ -64,6 +65,7 @@
 **/ios/ServiceDefinitions.json
 **/ios/Runner/GeneratedPluginRegistrant.*
 **/ios/Flutter/flutter_export_environment.sh
+**/ios/Podfile
```

Comment:
In upcoming flutter versions, it requires user to rm ios/Podfile and regenerates on build (with Podfile.lock kept), so I remove the file and gitignore it

Reply:
Could you point to a relevant discussion on flutter site so I could read more? I cannot google it

Reply:
@dariuszseweryn Ah... I don't think it's documented anywhere but if you switch flutter channel to beta or dev, then run cd example && flutter build ios --no-codesign .  it will warn that you are using an outdated Podfile, run rm ios/Podfile and build again